packages/venia-concept/src/components/Checkout/Receipt/receipt.css

```diff
@@ -38,3 +38,8 @@
     font-weight: 300;
     line-height: 1.25rem;
 }
+
+.button {
+    composes: root from 'src/components/Button/darkTheme.css';
```

Comment:
I'm confused as to why `receipt` uses css but the other components use the [darkThemeCss](https://github.com/magento-research/pwa-studio/pull/660/files#diff-ce60e415fd049caba4edf7d08e192fbcR5) export from button.